Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/Utils.h

```diff
@@ -28,5 +28,27 @@ namespace ImageProcessingAtom
         bool SaveImageToDdsFile(IImageObjectPtr image, AZStd::string_view filePath);
 
         bool NeedAlphaChannel(EAlphaContent alphaContent);
-    }
+
+        //! Load image assets in the background and execute callbacks when complete. 
+        class AsyncImageAssetLoader : public AZ::Data::AssetBus::MultiHandler
+        {
+        public:
+            using Callback = AZStd::function<void(const AZ::Data::Asset<AZ::RPI::StreamingImageAsset>&)>;
```

Comment:
nbd but it might be good to comment on whether the callback might happen on a thread other than the main thread